pytket/binders/circuit/library.cpp

```diff
@@ -41,6 +41,12 @@ void init_library(py::module &m) {
       "Given expressions α, β and γ, return circuit equivalent to "
       "TK2(α, β, γ) using up to 3 CX and single-qubit gates.\n\n"
       "The decomposition minimizes the number of CX gates.");
+  library_m.def(
+      "_TK2_using_CX_and_swap", &CircPool::TK2_using_CX_and_swap,
+      "Given expressions α, β and γ, return circuit equivalent to "
+      "TK2(α, β, γ), up to a wire swap, using up to 3 CX and single-qubit "
```

Comment:
Should probably make it clear here (and similarly below and in the documentation for the C++ functions) that the needed wire swap will be encoded in the implicit qubit permutation of the circuit; otherwise it looks like we may have lost that information.

Reply:
I've updated the definitions for `_TK2_using_CX_and_swap` and `TK2_using_ZZMax_and_swap`, and the comments describing the `CircPool` methods they use to express this